### PR TITLE
Create Dockerfile.ubuntu1804-horizon-5.2

### PR DESCRIPTION
**Pull request by mikaelkrantz945**
Suggest you add this one it has ubuntu 18.04 and missing libs for horizon client 5.2 that is latest from vmware. tested on ubuntu 18.04 server.

### CODE REVIEW

File: Dockerfile.ubuntu1804-horizon-5.2 [hunk 1]
```diff
@@ -0,0 +1,69 @@
+FROM ubuntu:18.04
+LABEL maintainer="exotime <exotime@users.noreply.github.com>"
+LABEL version="0.1.3"
+
+# Update this URL as necessary. One day I might work this into the script.
+# Run the following command to find the latest URL:
+# $ curl -s "https://my.vmware.com/web/vmware/details?downloadGroup=CART20FQ2_LIN64_510&productId=863&rPId=34529" | grep -o '<a href="http[^"]*.x64.bundle"' | sed 's/<a h
+ref="//;s/"$//'
```
Comment (exotime) [inline Dockerfile.ubuntu1804-horizon-5.2]: Hey, thanks for the PR! I think your editor may have broken line seven into two lines, preventing it from building.

Comment (mikaelkrantz945) [reply Dockerfile.ubuntu1804-horizon-5.2]: Ah yeah I did cut/paste into the github webgui. I can adjust it.

**Comment by exotime** [comment]
Thanks heaps for contributing! Hope the project is useful to you in your DaaS infrastructure.

**Comment by mikaelkrantz945** [comment]
Yeah dont forget to merge the run script also as the variable user IS needed.

Hämta Outlook för Android<https://aka.ms/ghei36>

________________________________

**Comment by exotime** [comment]
Yep, I see that. Have done already, thanks for the reminder!

**Pull request merged**